### PR TITLE
[#1] Merge Johnny's stuff into main

### PR DESCRIPTION
**Pull request by hocally**

### CODE REVIEW

File: data_entry/models.py [hunk 1]
```diff
@@ -3,7 +3,6 @@
 from django.contrib.auth.models import User
 from django.utils.timezone import now
 
-
```
Comment (hocally) [inline data_entry/models.py:6]: Nitpick: don't commit whitespace changes

File: data_entry/templates/data_entry/viewdata.html [hunk 2]
```diff
@@ -5,52 +5,166 @@
...
+					/*If a switch has been marked, make the switch
+					and mark that a switch has been done:*/
+					rows[i].parentNode.insertBefore(rows[i + 1], rows[i]);
+					switching = true;
+				}
+			}
+		}
+	</script>
 </body>
 </html>
```
Comment (hocally) [inline data_entry/templates/data_entry/viewdata.html:170]: Nit: files should end with newlines

File: data_entry/views.py [hunk 3]
```diff
@@ -176,35 +176,46 @@ def collectData(request):
...
+			}
+			entries = Test.objects.filter(**d)
+			context = {'entries': entries}
+		return render(request, 'data_entry/viewdata.html', context)
     # load all entries   
-    else:
-        entries = Test.objects.all()
-        context = {'entries': entries}
-        return render(request, 'data_entry/viewdata.html', context)
+	else:
```
Comment (hocally) [inline data_entry/views.py:215]: Nit: why change the indentation here?

File: static/css/viewdata.css [hunk 4]
```diff
@@ -2,15 +2,21 @@
...
 body { 
   margin: 0;
+  background-color: #862334;
   font-family: Arial, Helvetica, sans-serif;
 }
 
 .header {
   overflow: hidden;
-  background-color: #f1f1f1;
+  /* background-color: #f1f1f1; */
```
Comment (hocally) [inline static/css/viewdata.css:11]: Nit: this should be deleted

**Review by hocally — COMMENTED** [review]
Good job Johnny

**Review by jrajan15 — APPROVED** [review]
noice

**Pull request merged**